pkgpanda/actions.py

```diff
@@ -67,6 +68,18 @@ def swap_active_package(install, repository, package_id, systemd, block_systemd)
     activate_packages(install, repository, new_active, systemd, block_systemd)
 
 
+def show_package(repository, package_id):
```

Comment:
The functions in `actions` are intended to not be coupled to the application's interface, but `show_package()` prints its result, which means it's only useful in the context of a terminal. This function should return the `pkginfo` object and let the caller decide how to display it, but then it would do so little that it probably isn't worth defining as its own function.

Reply:
Agreed, inlined this into cli.py.